prime-dotnet-webapi/Services/EmailService.cs

```diff
@@ -27,19 +27,22 @@ private static class SendType
         private readonly IEmailDocumentsService _emailDocumentService;
         private readonly IEmailRenderingService _emailRenderingService;
         private readonly ISmtpEmailClient _smtpEmailClient;
+        private readonly IEnrolleeService _enrolleeService;
 
         public EmailService(
             ApiDbContext context,
             ILogger<EmailService> logger,
             IChesClient chesClient,
             IEmailDocumentsService emailDocumentService,
             IEmailRenderingService emailRenderingService,
+            IEnrolleeService enrolleeService,
             ISmtpEmailClient smtpEmailClient)
             : base(context, logger)
         {
             _chesClient = chesClient;
             _emailDocumentService = emailDocumentService;
             _emailRenderingService = emailRenderingService;
+            _enrolleeService = enrolleeService;
```

Comment:
IMO two options here:
1. `EnrolleeService` is no longer needed and removed. Minimal changes needed.
2. Keep the injected `EnrolleeService` and move the entire`getRenewalEnrollees` business logic into `EnrolleeService`. `getRenewalEnrollees` returns a list of `EnrolleeRenewalEmailViewModel` with an added Email field.
Advantage of `#2` is separation of different business.

Reply:
I was actually using `GetNonAbsentEnrollees` in `EnrolleeService` which returns a list of enrollee ids of those who are not absent (similar to the query in the renewal method) then I was calling the `!Contains` on that list, I forgot to remove the `EnrolleeService` injection afterwards. The reason I removed it from EnrolleeService was because we don't have a use for that list outside the renewal emails, I'm open to using EnrolleeService again, or we can save that for the future in case we ever need to use that list somewhere else.  For now I just removed the service.